src/kcm/ExceptionListWidget.cc

```
@@ -0,0 +1,298 @@
+    const QModelIndex index = m_listView->currentIndex();
+    if (!index.isValid()) {
+        return;
+    }
+
+    m_model->remove(index.row());
+    updateButtons();
+}
+
+void ExceptionListWidget::up()
+{
+    const QModelIndex index = m_listView->currentIndex();
```

Clang (14.0.6)

[warning] variable 'index' is not initialized

(cppcoreguidelines-init-variables)